journal.py

```diff
@@ -41,11 +59,31 @@ class Entry(Base):
     def write(cls, title=None, text=None, session=None):
         if session is None:
             session = DBSession
+        text = style_text(text)
```

Comment:
hmmm.  This is not going to work well for you.  Think about it.  Markdown is a one-way transform.  You change text into HTML, but you can't change the HTML back into the text it came from.  So if you're storing the HTML in the database, it will not be editable again as 'markdown'.